ci/axis/gpu.yml

```
@@ -0,0 +1,23 @@
+# Copyright (c) 2018-2020 NVIDIA Corporation
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+# Released under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+
+SDK_TYPE:
+  - nvcr.io/nvidia/cuda
+
+SDK_VER:
+  - 11.1-devel
+
+OS_TYPE:
+  - ubuntu
+
+OS_VER:
+  - 20.04-devel
```

This doesn't match cccl: https://github.com/NVIDIA/cccl/blob/main/ci/axis/docker.yml#L19
```suggestion
  - 20.04
```

---

Indeed it does not. Fixed.